runtime/tests/staking_reward.rs

```diff
@@ -547,11 +562,15 @@ fn slashed_cennz_goes_to_reporter() {
 			};
 
 			let slash_fraction = Perbill::from_percent(90);
-			assert_ok!(Staking::on_offence(
-				&[offence],
-				&[slash_fraction],
-				Staking::eras_start_session_index(active_era()).expect("session index exists"),
-			));
+			assert_eq!(
+				Staking::on_offence(
+					&[offence],
+					&[slash_fraction],
+					Staking::eras_start_session_index(active_era()).expect("session index exists"),
+					DisableStrategy::Never,
```

Comment:
```suggestion
					DisableStrategy::WhenSlashed,
```
will probably fix the test